_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -52,7 +53,9 @@ For this rule, it is assumed that the `autocomplete` attribute is not used on fo
 
 ## Accessibility Support
 
-While `autocomplete` in a promising technique for supporting personalization in HTML, support for this is fairly limited.
+- While `autocomplete` in a promising technique for supporting personalization in HTML, support for this is fairly limited.
```

Comment:
```suggestion
- Support for the `autocomplete` technique is fairly limited.
```
Just to remain as objective as possible. I don't see how "promising" is relevant here.

Reply:
I think this is a separate topic and should be part of another PR if needed. Would you like to handle it, @daniel-montalvo ?